dev/releases/release_notes.py

```diff
@@ -82,7 +82,7 @@ def warning(s):
     ["package: Nemo", "Changes related to the package Nemo"],
     ["package: Polymake", "Changes related to the package Polymake"],
     ["package: Singular", "Changes related to the package Singular"],
-    ["renaming", "Items being renamed ?"],
+    ["renaming", "Items being renamed"],
     ["serialization", "Changes related to serializing data in the MRDI file format ?"],
     ["topic: algebraic geometry", "Changes related to Algebraic Geometry"],
```

Comment:
Regarding this list, I think what I really would want is this:
1. first all highlights
2. all renaming or removals (aka "future breaking changes")
2. then a list of all topics
   - under each topic, subsections for "enhancements", "bug fixes", etc.
3. a section "Miscellaneous" with the rest (?)